index.html

```diff
@@ -172,5 +172,29 @@ <h3 class="about__links">LET'S CONNECT</h3>
         <hr>
     </div>
   </section>
+  <section class="contact_form">
+    <div class="contact__form">
+        <h2 class="contact__title">Contact me</h2>
+        <p class="contact__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    </div>
+    <form action="https://formspree.io/f/xgebklbg" method="post" class="form__list">
```

Comment:
- [ ] You got this right, using the correct form action & method is very important for the correct functioning of the form 👏🏾